lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -3890,30 +3893,26 @@ namespace Js
 
         if (scriptConfig->IsES6RegExPrototypePropertiesEnabled())
         {
-            RuntimeFunction* globalGetter = library->CreateGetterFunction(PropertyIds::global, &JavascriptRegExp::EntryInfo::GetterGlobal);
-            library->AddAccessorsToLibraryObject(regexPrototype, PropertyIds::global, globalGetter, nullptr);
-            library->regexGlobalGetterFunction = globalGetter;
-
+            library->regexGlobalGetterFunction =
```

Comment:
Current function InitializeRegexPrototype should have passed right initSlotCapacity (typeHandler->Convert(, , 8) )?